tools/osbuilder/image-builder/image_builder.sh

```diff
@@ -476,6 +476,80 @@ and the rootfs is built with SELINUX=yes"
 	rmdir "${mount_dir}"
 }
 
+create_erofs_rootfs_image() {
```

Comment:
I stripped down the repetitive code.
However, since the construction process of EROFS is different from that of ext4 and xfs, too much redundant judgment logic will be added in the same method, and `create_erofs_rootfs_image()` is still used alone.

Reply:
I encapsulated the operations of `systemd` and `SELinux` into `setup_systemd()` and `setup_selinux()` methods.